module.config.js.example

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  /*
+  Modules you want to use from local source code.  Adding a module here means that when this app
+  runs its build, it'll resolve the source from peer directories of this app.
+
+  moduleName: the name you use to import code from the module.
+  dir: The relative path to the module's source code.
+  dist: The sub-directory of the source code where it puts its build artifact.  Often "dist".
+
+  To use a module config:
+
+  1. Copy module.config.js.example and remove the '.example' extension
+  2. Uncomment modules below in the localModules array to load them from local source code.
+  3. Remember to re-build the production builds of those local modules if they have one.
+     See note below.
+  */
+  localModules: [
+    /*********************************************************************************************
+     IMPORTANT NOTE: If any of the below packages (like paragon or frontend-platform) have a build
+     step that populates their 'dist' directories, you must manually run that step.  For paragon
+     and frontend-platform, for instance, you need to run `npm run build` in the repo before
+     module.config.js will work.
+    **********************************************************************************************/
+
+    // { moduleName: '@edx/brand', dir: '../brand-openedx' }, // replace with your brand checkout
+    // { moduleName: '@edx/paragon/scss/core', dir: '../paragon', dist: 'scss/core' },
+    // { moduleName: '@edx/paragon/icons', dir: '../paragon', dist: 'icons' },
+    // { moduleName: '@edx/paragon', dir: '../paragon', dist: 'dist' },
```

Comment:
We should make clear to the team that adding paragon to the module config is only needed if you are working on contributing to paragon. (The purpose being to see your paragon component-in-progress inside your app rather than the paragon workbench).

If people uncomment this they often run into needing to pull and build the new version of paragon to do unrelated squad work.